packages/protocol/contracts/governance/Validators.sol

```diff
@@ -708,13 +734,21 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
     uint256[] memory numMembersElected = new uint256[](electionGroups.length);
     uint256 totalNumMembersElected = 0;
     bool memberElectedInRound = true;
+
+    uint256 votesTotal = 0;
+    for (uint256 i = 0; i < electionGroups.length; i = i.add(1)) {
+      votesTotal = votesTotal.add(votes.getValue(electionGroups[i]));
```

Comment:
`electionGroups` will omit some groups that received votes. Is this okay?

Reply:
Well I don't think it's okay. So I'll have another storage variable for keeping track of vote total.